packages/aura/src/components/button.css

```diff
@@ -1,3 +1,33 @@
-html {
-  --vaadin-button-background: light-dark(var(--vaadin-background-dark), var(--vaadin-background-light));
+vaadin-button {
+  --vaadin-button-border-color: var(--vaadin-border-color);
+  --aura-surface-level: 4;
+  box-shadow: 0 1px 4px -1px hsla(0, 0%, 0%, 0.07);
+  transition:
+    scale 220ms,
+    filter 220ms;
+}
+
+vaadin-button:not([disabled], [aria-disabled='true'])[active] {
+  scale: 0.98;
+  transition-duration: 50ms;
+  filter: brightness(0.95);
+}
+
+/*
+Increase padding, but only for buttons that don't have an icon in the default slot.
+Buttons that place an icon in the default slot are assumed to be icon-only buttons.
+*/
+vaadin-button:not(:has(vaadin-icon:not([slot]))) {
```

Comment:
Just a note: we use no vaadin-icon inside the button, so it might be better to broaden this selector if people e.g. use svg directly or fontawesome's i (font) integration